src/service_router.rs

```diff
@@ -4,7 +4,7 @@
 
 use context::{ ContextTrait, Shared };
 use iron::{Request, Response, IronResult};
-use iron::headers::ContentType;
+use iron::headers::{ ContentType, AccessControlAllowOrigin };
```

Comment:
Are the changes in `service_router.rs` intended for the origin of the PR? Or maybe I'm missing something.